src/components/map/Popups/StudyPopup.tsx

```diff
@@ -43,10 +62,10 @@ export default function StudyPopup(record: AirtableRecord) {
             {`${record.denominator}`}
         </div>
         <div className="popup-heading">
-            {Translate("PopulationGroup")}
+            {Translate("VaccineRolloutStatus")}
         </div>
         <div className="popup-text">
-            {record.population_group ? `${record.population_group}` : Translate("NotReported")}
+            {(record.vaccination_policy == 0) ? "Vaccines unavailable" : (record.vaccination_policy == 1 || record.vaccination_policy == 2 || record.vaccination_policy == 3) ? "Vaccines available to certain groups" : "Vaccines available to broader population"}
```

Comment:
Doing this out of JSX and in a variable above could make this more clear / readable

Reply:
you mean turn this into a set of if statements? what do you mean by extracting into another variable and what do you mean by magic string?

Reply:
These strings should be translated!!

Reply:
done